### PR TITLE
NeCombobox: support for multiple selection

### PR DESCRIPTION
**Pull request by andre8244**
Other changes:
- `NeBadge`: add `iconClickable` prop

### CODE REVIEW

File: src/components/NeBadge.vue [hunk 1]
```diff
@@ -128,17 +163,27 @@ const typeClasses = computed(() => {
...
-    <FontAwesomeIcon
-      v-if="icon"
-      :class="[iconPosition == 'right' ? 'order-1' : '']"
-      :icon="icon"
-      class="h-4 w-4"
-    />
+    <template v-if="icon">
+      <button
+        v-if="iconClickable"
+        @click="onIconClick"
```
Comment (Tbaile) [inline src/components/NeBadge.vue:174]: I would've rather put the emit here (removing the function), but at your discretion

**Review by Tbaile — APPROVED** [review]
Don't really like the loosen types of the ComboBox, but since it works :man_shrugging:

File: src/components/NeBadge.vue (hunk 1 above)
Comment (andre8244) [reply src/components/NeBadge.vue:174]: I try to remove logic from the template whenever possibile, but this is a borderline case and it's more a matter of taste :man_shrugging:

**Comment by andre8244** [comment]
> Don't really like the loosen types of the ComboBox, but since it works 🤷‍♂️

Good point, I'm going to improve this

**Pull request merged**